web/src/api/signup/index.ts

```diff
@@ -116,36 +128,113 @@ export const handleSignup = withApiAuthRequired(
         auth0Id: auth0User.sub,
```

Comment:
add name and email here

Reply:
@andy-t-wang I've added name with default `""` value and nullable email value. The thing is that when user accepts invite, link from email redirect to `/login?invite_id=blabla`. and then user presses the login button to login/signup and then user can pick email or worldcoin auth method. In case worldcoin auth there will be no such data as name or email. Only `nullifier_hash`